app/src/main/java/com/sunasterisk/a14day_challenge/ui/process/ProcessPresenter.kt

```diff
@@ -0,0 +1,25 @@
+package com.sunasterisk.a14day_challenge.ui.process
+
+import com.sunasterisk.a14day_challenge.data.UserRepository
+
+class ProcessPresenter(
+    var view: ProcessContract.View,
+    private val userRepository: UserRepository
```

Comment:
Nếu chỉ có 1 repository, thì chỉ cần đặt tên là repository thôi 
```suggestion
    private val repository: UserRepository
```